packages/angular-workspace/package.json

```diff
@@ -25,29 +25,29 @@
     "format": "ng lint --fix"
   },
   "dependencies": {
-    "@angular/animations": "^16.2.12",
-    "@angular/common": "^16.2.12",
-    "@angular/compiler": "^16.2.12",
-    "@angular/core": "^16.2.12",
-    "@angular/forms": "^16.2.12",
-    "@angular/platform-browser": "^16.2.12",
-    "@angular/platform-browser-dynamic": "^16.2.12",
-    "@angular/router": "^16.2.12",
+    "@angular/animations": "^17.3.11",
+    "@angular/common": "^17.3.11",
+    "@angular/compiler": "^17.3.11",
+    "@angular/core": "^17.3.11",
+    "@angular/forms": "^17.3.11",
+    "@angular/platform-browser": "^17.3.11",
+    "@angular/platform-browser-dynamic": "^17.3.11",
+    "@angular/router": "^17.3.11",
     "@ni/nimble-components": "*",
     "@ni/spright-components": "*",
     "rxjs": "^7.3.0",
     "tslib": "^2.2.0",
-    "zone.js": "^0.13.0"
+    "zone.js": "^0.14.0"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^16.2.12",
-    "@angular/cli": "^16.2.12",
-    "@angular/compiler-cli": "^16.2.12",
-    "@angular/localize": "^16.2.12",
+    "@angular-devkit/build-angular": "^17.3.8",
+    "@angular/cli": "^17.3.8",
+    "@angular/compiler-cli": "^17.3.11",
+    "@angular/localize": "^17.3.11",
     "@lhci/cli": "^0.13.0",
     "@microsoft/fast-web-utilities": "^6.0.0",
     "@ni-private/eslint-config-nimble": "*",
-    "@ni/eslint-config-angular": "^7.0.6",
+    "@ni/eslint-config-angular": "../../ni-eslint-config-angular-8.0.0.tgz",
```

Comment:
Reminder to replace this and regen the lock file once the actual package is published.